components/SliderPlusDescription.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-carousel>
+    <v-carousel-item
+      v-for="(slide, index) in slides"
+      :key="index"
+      :src="item.src"
+      cover
+    >
+      <div class="caption">
+        <h2>{{ slide.caption.header }}</h2>
+        <p>{{ slide.caption.intro }}</p>
+      </div>
+    </v-carousel-item>
+  </v-carousel>
+</template>
+
+<script>
+export default {
```

Comment:
@AgnieszkaIwan could you use <script setup> for consistency with the rest of a project?